docs/graphql/manual/guides/deployment/kintohub-one-click.rst

```diff
@@ -0,0 +1,61 @@
+.. meta::
+   :description: Deploy Hasura GraphQL engine in one-click on KintoHub
+   :keywords: hasura, docs, guide, deployment, kintohub, one-click
+
+Hasura GraphQL engine in one-click on KintoHub
+==============================================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+The Hasura GraphQL engine is available in one-click on KintoHub. It is packed with a private `Postgres <https://www.postgresql.org/>`__ instance and secured with a free and automatic HTTPS certificate using `Let's Encrypt <https://letsencrypt.org/>`__.
+
+Quickstart
+----------
+
+1. Deploy your private Hasura and Postgres Database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Click the button below to deploy your Hasura GraphQL engine and Postgres instances in your private environment. (``Ctrl+Click`` to open in a new tab)
```

Comment:
What does "in your private environment" mean?

Reply:
Added a link to our doc about environment.